WalletWasabi.Fluent/ViewModels/Wallets/Send/PrivacyControlViewModel.cs

```diff
@@ -73,6 +75,7 @@ public PrivacyControlViewModel(Wallet wallet, TransactionInfo transactionInfo)
 		private async Task OnNextAsync(TransactionInfo transactionInfo,
 			IObservableList<PocketViewModel> selectedList)
 		{
+			_buildingTransaction = true;
```

Comment:
This has to be inside the try. The pattern is `try{ flag = whatever; } finally { flag = original_value; }`